Makefile

```diff
@@ -1,57 +1,46 @@
-PKGS ?= $(shell glide nv)
-PKG_FILES ?= *.go
-
-# The linting tools evolve with each Go version, so run them only on the latest
-# stable release.
-GO_VERSION := $(shell go version | cut -d " " -f 3)
-GO_MINOR_VERSION := $(word 2,$(subst ., ,$(GO_VERSION)))
-LINTABLE_MINOR_VERSIONS := 12
-ifneq ($(filter $(LINTABLE_MINOR_VERSIONS),$(GO_MINOR_VERSION)),)
-SHOULD_LINT := true
-endif
-
-.PHONY: dependencies
-dependencies:
-	@echo "Installing Glide and locked dependencies..."
-	glide --version || go get -u -f github.com/Masterminds/glide
-	glide install
-ifdef SHOULD_LINT
-	@echo "Installing golint..."
-	go get -u golang.org/x/lint/golint
-else
-	@echo "Not installing golint, since we don't expect to lint on" $(GO_VERSION)
-endif
+export GOBIN ?= $(shell pwd)/bin
+
+GO_FILES := $(shell \
+	find . '(' -path '*/.*' -o -path './vendor' ')' -prune \
+	-o -name '*.go' -print | cut -b3-)
+
+GOLINT = $(GOBIN)/golint
+STATICCHECK = $(GOBIN)/staticcheck
+
+.PHONY: build
+build:
+	go build ./...
+
+.PHONY: install
+install:
+	go mod download
 
 .PHONY: test
 test:
-	@.build/test.sh
+	go test -race ./...
 
-# Disable printf-like invocation checking due to testify.assert.Error()
-VET_RULES := -printf=false
+.PHONY: cover
+cover:
+	go test -race -coverprofile=cover.out -covermode=atomic -coverpkg=./... ./...
```

Comment:
do we need `-race` for cover? since we already use `-race` in the `test` rule above

Reply:
You're right, we don't.